flask_appbuilder/tests/test_fab_cli.py

```diff
@@ -70,3 +75,46 @@ def test_list_views(self):
             result = runner.invoke(list_views, [])
             self.assertIn("List of registered views", result.output)
             self.assertIn(" Route:/api/v1/security", result.output)
+
+    def test_import_export_roles(self):
+        with tempfile.TemporaryDirectory() as tempdir:
+            src_app = Flask("src_app")
+            src_app.config.from_object("flask_appbuilder.tests.config_security")
+            src_app.config[
+                "SQLALCHEMY_DATABASE_URI"
+            ] = f"sqlite:///{os.path.join(tempdir, 'src.db')}"
+            src_db = SQLA(src_app)
+            src_app_builder = AppBuilder(src_app, src_db.session)
+            src_cli_runner = src_app.test_cli_runner()
+
+            dst_app = Flask("dst_app")
+            dst_app.config.from_object("flask_appbuilder.tests.config_security")
+            dst_app.config[
+                "SQLALCHEMY_DATABASE_URI"
+            ] = f"sqlite:///{os.path.join(tempdir, 'dst.db')}"
+            dst_app.config["FAB_ROLES"] = {}
+            dst_db = SQLA(dst_app)
+            dst_app_builder = AppBuilder(dst_app, dst_db.session)
+            dst_cli_runner = dst_app.test_cli_runner()
+
+            filename = "roles.json"
+
+            export_result = src_cli_runner.invoke(export_roles, [f"--path={filename}"])
+            self.assertEqual(export_result.exit_code, 0)
+            self.assertTrue(os.path.exists(filename))
```

Comment:
Would be great if we could assert the content of the file also.
nit: can you split these into 2 distinct tests? by assert the content you can add the file has a fixture